Learn-CSS-Blog/index31.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+    <div id="main">
+        <header>
+            <nav>
+                <ul class="nav-links">
+                    <li><a href="#">Intro</a></li>
+                    <li><a href="#">Blog</a></li>
+                </ul>
+                <a href="#" id="logo" >
+                    <img src="https://d33wubrfki0l68.cloudfront.net/e66392bc2941ad564b6956321e778f29073cf585/4190b/img/glyph-alternative.svg" alt="freeCodeCamp logo">
+                </a>
+            </nav>
+        </header>
+        <section class="container">
+            <div class="about">
+                <div>
+                    <img class="img" src="img/profile.jpg" alt="profile picture">
+                </div>
+                <div class="about-content"></div>
+            </div>
+        </section>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Nest a `p` element inside the `div` with the words `Hello World!`.
```

Comment:
Nest a `<p>` element inside the "about-content" `<div>` with the text "Hello World!".